kivymd/uix/behaviors/elevation.py

```diff
@@ -183,37 +171,98 @@ class CommonElevationBehavior(object):
     """
 
     _elevation = NumericProperty(0)
+
+    # soft shadow
     _soft_shadow_texture = ObjectProperty()
     _soft_shadow_size = ListProperty((0, 0))
     _soft_shadow_pos = ListProperty((0, 0))
+    _soft_shadow_cl = ListProperty([1, 1, 1, 0])
     _soft_shadow_a = NumericProperty(0)
+
+    # hard shadow
     _hard_shadow_texture = ObjectProperty()
     _hard_shadow_size = ListProperty((0, 0))
     _hard_shadow_pos = ListProperty((0, 0))
+    _hard_shadow_cl = ListProperty([1, 1, 1, 0])
     _hard_shadow_a = NumericProperty(0)
 
     def __init__(self, **kwargs):
+        #
+        im = BytesIO()
+        Image.new("RGBA", (4, 4), color=(0, 0, 0, 0)).save(im, format="png")
+        im.seek(0)
+        #
+        self._soft_shadow_texture = self._hard_shadow_texture = CoreImage(
+            im, ext="png"
+        ).texture
+        super().__init__(**kwargs)
+        Clock.schedule_once(self.shadow_preset, -1)
+
+    def shadow_preset(self, *dt):
+        """
+        This function is meant to set the deffault configuration of the
+        elevation.
+
+        After a new instance is created, the elevation property will be launched
+        and thus this function will update the elevation if the KV lang have not
+        done it already.
+
+        Works similar to an `__after_init__` call inside a widget.
+        """
+        if self.elevation is None:
+            self.elevation = 10
+        self._update_shadow(None, self.elevation)
         self.bind(
-            elevation=self._update_elevation,
             pos=self._update_shadow,
             size=self._update_shadow,
         )
-        super().__init__(**kwargs)
```

Comment:
This function works similar to an __after_init__ call, it will be called before the next frame and will have all the kvlang and **kwargs alredy set on the propertyes. so it will know if a value is None and set to it's deffault.